modules/coxBidAdapter.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+import * as utils from 'src/utils';
+import { BANNER } from 'src/mediaTypes';
+import { config } from 'src/config';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const helper = (() => {
+  let srTestCapabilities = () => { // Legacy
+    let plugins = navigator.plugins;
+    let flashVer = -1;
+    let sf = 'Shockwave Flash';
+
+    if (plugins && plugins.length > 0) {
+      if (plugins[sf + ' 2.0'] || plugins[sf]) {
+        var swVer2 = plugins[sf + ' 2.0'] ? ' 2.0' : '';
+        var flashDescription = plugins[sf + swVer2].description;
+        flashVer = flashDescription.split(' ')[2].split('.')[0];
+      }
+    }
+    if (flashVer > 4) return 15; else return 7;
+  };
+
+  let getRand = () => {
+    return Math.round(Math.random() * 100000000);
+  };
+
+  // State variables
+  let env = '';
```

Comment:
These variables are acting as global variables. 

Bidder Requirements:
_Support multiple instances: All adapters must support the creation of multiple concurrent instances. This means, for example, that adapters cannot rely on mutable global variables._